RetailSuiteAccounting/Operation/10_ManageReceipts.md

```diff
@@ -0,0 +1,194 @@
+
+1. Select the appropriate currency in the *Currency code* field, if necessary.
+
+  > [Info] Currency exchange rate can also be adjusted if needed in the *Rate* field.
+
+2. Enter the total invoice value in the *Turnover* field.
+
+3. Enter an account number in the *Contra Account* field.
+
+4. Enter the date for the manual booking in the *Date* field. Date format must be DDMMYY or DD.MM.YY.
+
+5. Enter the creditor (supplier) account number in the *Account* field.
+
+  > [Info] Note that the fields *Contra Account*, *Date*,  *Account* must be filled out to be able to book a receipt.
+
+6. Enter any other relevant booking information in the corresponding fields.
+
+  > [Info] Even though it is not compulsory, it is highly recommended to fill in the *Receipt 1* field, as the system identifies and groups together any subsequent bookings taking this field as reference.
+
+  ![Booking a receipt](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/NewDocument01.png "[Booking a receipt]")
+
+7. Click the [BOOK] button.  
+A message above the entry line confirms that the receipt has been linked to the booking. A new booking is displayed in the bookings list in the *BOOKING* tab. A ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon (=Beleg, EN: receipt) is displayed next to the booking in the bookings list.
+
+  ![Receipt linked](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/DocumentLinked01.png "[Receipt linked]")
+
+  > [Info] If there are further unregistered external receipts, they will be displayed in the *BOOK RECEIPTS* tab. Otherwise, the message "No documents left to be registered" is displayed.
+
+
+## Book a supplier receipt
+
+The *Supplier receipts* menu entry of the *EDIT* context menu in the *BOOKINGS* is an enhanced function to book specifically supplier receipts.
+
+Suppliers' invoices can be uploaded in PDF format in the *New supplier receipts* folder in the *Documents* module, which will be then recognized by the system and displayed in the *Acquire* tab in the *Supplier receipts* window. Once they are booked, they are moved automatically into the *Acquired documents* folder.
+
+ Moreover, this function interacts with the *Purchasing* and *Payment processing* modules, and allows to check supplier receipts before booking them in the system and to automatize payment conditions.
+
+[comment]: <> (ACQUIRE probably "enter" -ERFASSEN- or "park" -vorerfassen", SAP Terminology)
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
+- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
+- The appropriate plug-in is installed (Checking of supplier receipts + payment proposal list).
+- The corresponding folder is created in the *Documents* module.
+- Supplier receipts are uploaded in the *New supplier receipts* folder.
```

Comment:
```suggestion
- A fiscal year has been selected, see [Select fiscal year](./01_SelectFiscalYear.md).
- The *Supplier receipts + payment proposal list* plug-in has been installed.
- At least one supplier receipt has been uploaded in the *New supplier receipts* folder.
```
Was meinst du mit dem Punkt "The corresponding folder is created in the *Documents* module."? Der New supplier receipts Ordner wird automatisch erstellt. Welcher Ordner wird sonst noch benötigt?

Reply:
Ich habe den Ordner selber erstellen müssen. Es war bei mir nicht vorhanden. Wann ist es automatisch erstellt?